composer.json

```diff
@@ -3,13 +3,15 @@
 	"license": "GPL-2.0-or-later",
 	"description": "Bounded Context for the Wikimedia Deutschland fundraising membership subdomain",
 	"require": {
-		"php": ">=7.3",
+		"php": ">=7.4",
+
+		"doctrine/dbal": "^2.7",
```

Comment:
dbal should be an of implicit dependency of `doctrine/orm`, I only put it in during debugging